service-bus/router/src/lib.rs

```diff
@@ -182,16 +182,20 @@ where
             }
         } else {
             match self.registered_endpoints.entry(msg.service_id.clone()) {
-                Entry::Occupied(_) => RegisterReply {
-                    code: RegisterReplyCode::RegisterConflict as i32,
-                    message: format!("Service ID '{}' already registered", msg.service_id),
-                },
+                Entry::Occupied(_) => {
+                    log::warn!("Service '{}' already registered", msg.service_id);
```

Comment:
Won't this effectively print the message twice (for local router)?

Reply:
Indeed. Reverting this.